### PR TITLE
Refactor/minor function fixes

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Rewrote functions written in ES5 to ES6 in recipesUtil.js.
- Leveraged truthy/falsy javascript principles in functions that were not doing so.

Did this break anything?
- [x] No
- [ ]  Yes

Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Improving UI in app.

### CODE REVIEW

**Review by hvohr — APPROVED** [review]
I love the use of truthy and falsy leveraging here to dry up the code

File: src/domUpdates.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ function showSearchResults() {
   allRecipeDisplay.innerHTML = '';
   getData('recipes').then(({ recipes }) => {
     const searchedRecipes = recipesfromName(recipes, searchValue);
-    if (searchedRecipes.length === 0) {
+    if (!searchedRecipes.length) {
```
Comment (andreasorensen) [inline src/domUpdates.js:148]: Thanks for fixing the truthy/falsey!

File: src/recipeUtils.js [hunk 2]
```diff
@@ -8,7 +7,7 @@ function recipesFromTag(recipes, tags) {
   return filtered
 }
 
-function recipesfromName(recipes, name) {
+const recipesfromName = (recipes, name) => {
```
Comment (andreasorensen) [inline src/recipeUtils.js:10]: Love ES6!

**Review by andreasorensen — APPROVED** [review]
Excellent job with this refactor.

**Review by Denzel-Turambi — APPROVED** [review]

**Pull request merged**